AdaSupport/embed
# INT-319: Introduce Drag and Drop to Embed

### What
* Introduce `dragAndDrop` option to embed
* If turned on Button & Intros will be draggable
* Button/Intro will not active during drag
### Test
* Load embed and drag it around
* Open & Close Chat should still will be where it was left
---
